src/AppBundle/Entity/TaskCollectionItem.php

```diff
@@ -11,8 +11,9 @@
  *
  * @ORM\Entity
  * @ORM\Table(name="task_collection_item", uniqueConstraints={
- *   @ORM\UniqueConstraint(name="task_collection_item_unique", columns={"parent_id", "task_id"})}
- * )
+ *   @ORM\UniqueConstraint(name="task_collection_item_unique", columns={"parent_id", "task_id"}),
```

Comment:
As a side-note I think if we had separate models for `Delivery` and `TaskList` we would have a single unique constraint on `task_id` which will make things simpler in the long run (task can only be assigned once to a courier will be enforced at the database level).